Apps/AdminWebClient/src/ClientApp/src/views/Communication.vue

```diff
@@ -1,59 +1,69 @@
 <template>
-  <v-container>
-    <LoadingComponent :is-loading="isLoading"></LoadingComponent>
-    <BannerFeedbackComponent
-      :show-feedback.sync="showFeedback"
-      :feedback="bannerFeedback"
-      class="mt-5"
-    ></BannerFeedbackComponent>
-    <v-form ref="form" v-model="valid">
-      <v-row>
-        <v-col md="9">
-          <v-row>
-            <v-col>
-              <v-text-field
-                v-model="subject"
-                label="Subject"
-                maxlength="100"
-                :rules="[v => !!v || 'Subject is required']"
-                required
-              ></v-text-field>
-            </v-col>
-          </v-row>
-          <v-row>
-            <v-col>
-              <v-textarea
-                v-model="text"
-                label="Communication Text"
-                maxlength="1000"
-                :rules="[v => !!v || 'Text is required']"
-                required
-              ></v-textarea>
-            </v-col>
-          </v-row>
-          <v-row>
-            <v-col>
-              <v-datetime-picker
-                v-model="effectiveDateTime"
-                requried
-                label="Effective On"
-              ></v-datetime-picker>
-            </v-col>
-            <v-col>
-              <v-datetime-picker
-                v-model="expiryDateTime"
-                required
-                label="Expires On"
-              ></v-datetime-picker>
+    <v-container>
```

Comment:
seems like your code formatter is formatting things weirdly too. check in after standup and we can get your environment formatting properly :)